js/piwik.js

```diff
@@ -2605,10 +2609,34 @@ if (typeof Piwik !== 'object') {
                 callback();
             }
 
+            /**
+             * Send each bulk tracks to the server
+             * @return {void}
+             */
+            function sendBulkTrackingRequests() {
+
+                // Work on a local copy
+                var tmp = JSON.parse(JSON.stringify(bulkTrackingCollection));
+                bulkTrackingCollection.length = 0;
```

Comment:
Do you mind writing `bulkTrackingCollection = []`? There's nothing wrong with `bulkTrackingCollection.length = 0` as it seems to work fine but `bulkTrackingCollection = []` might be easier to read and understand? I had no idea it was possible to clear an array that way and had to test it first what it does. Setting a new empty array is maybe more clear